superset-frontend/src/components/FilterableTable/FilterableTable.tsx

```diff
@@ -337,9 +342,9 @@ export default class FilterableTable extends PureComponent<
         return -1;
       }
       if (descending) {
-        return aValue < bValue ? 1 : -1;
+        return parseFloatingNums(aValue) < parseFloatingNums(bValue) ? 1 : -1;
```

Comment:
Ah geez, it's not sorting strings correctly now. I thought that `value.isNan ? value : parseFloat(value)` would process strings properly since it checks if the value isn't a number and returns it unparsed if it's not. Good catch, thanks! I'll look further into this.

Reply:
Another thing: I don't think `isNaN` exists the way you're using it? AFAIK there's a global function `isNaN()`, and also another function in `Number.isNaN()`.

Reply:
Thank you for the tips! As discussed in our 1:1, I've got the sort now working with strings _and_ numbers. I also got it to work with localeCompare, it's looking cleaner and now it works multilingually! All done in [`this commit`](https://github.com/apache/superset/pull/17573/commits/43742de0d792866aa22734ee41a4e82bbcb20643) 😁